pytorch_lightning/accelerators/gpu.py

```diff
@@ -30,6 +30,9 @@
 class GPUAccelerator(Accelerator):
     """Accelerator for GPU devices."""
 
+    def __init__(self, benchmark=False):
+        self._benchmark = benchmark
```

Comment:
```suggestion
    def __init__(self, benchmark: bool = False):
        self._benchmark: bool = benchmark
```

Reply:
```suggestion
    def __init__(self, benchmark: bool = False):
        self._benchmark = benchmark
```

Only one annotation is required 😉